src/modal/modal-window.ts

```diff
@@ -32,6 +32,7 @@ import {ngbRunTransition, NgbTransitionOptions} from '../util/transition/ngbTran
     '[attr.aria-modal]': 'true',
     '[attr.aria-labelledby]': 'ariaLabelledBy',
     '[attr.aria-describedby]': 'ariaDescribedBy',
+    '(click)': 'bump($event)'
```

Comment:
I also just realised while adding the comment about the <kbd>Esc</kbd> key, that it might be better to add the bump call in the `_enableEventHandling() → fromEvent<MouseEvent>(nativeElement, 'click')` ?